<launch>
    <arg name="start_hardware_interface" default="false"/>

    <rosparam file="$(find robopi_low_level)/config/control.yaml" command="load"/>
    <rosparam file="$(find robopi_low_level)/config/sonar.yaml" command="load"/>
    <rosparam file="$(find robopi_low_level)/config/wheels.yaml" command="load"/>

    <node name="low_level_control_spawner" pkg="controller_manager" type="spawner"
          respawn="true" output="screen"
          args="/robopi/joint_state_controller
                /robopi/diff_drive_controller"/>

    <group if="$(eval arg('start_hardware_interface') == true)">
        <arg name="node_start_delay" default="5.0" />
        <node name="hardware_interface" pkg="robopi_low_level" type="hardware_interface_node" output="screen"/>
     <!--   <node name="sonar" pkg="robopi_low_level" type="sonar_node" output="screen"/> -->
    </group>


</launch>




